tests/conformance/state/state.go

```diff
@@ -0,0 +1,202 @@
+package state
+
+import (
+	"encoding/json"
+	"fmt"
+	"sort"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/dapr/components-contrib/state"
+	"github.com/dapr/components-contrib/tests/conformance/utils"
+	"github.com/stretchr/testify/assert"
+)
+
+type ValueType struct {
+	Message string `json:"message"`
+}
+
+const (
+	defaultDuration          = 10 * time.Millisecond
+	defaultBulkdRequestCount = 10
+)
+
+type TestConfig struct {
+	component         string
+	allOperations     bool
+	operations        []string
```

Comment:
If you intend this to be a set, I'd use `https://godoc.org/k8s.io/apimachinery/pkg/util/sets#NewString`, which already has the `Contains` and you won't need to do sorting.

I wouldn't worry about adding dependency here because it's for tests and won't increase our binary size, etc.

Reply:
done